proto/observer/params.proto

```diff
@@ -19,11 +20,21 @@ message CoreParams {
   string zeta_token_contract_address = 8;
   string connector_contract_address = 9;
   string erc20_custody_contract_address = 10;
-  int64 chain_id = 11;
   int64 outbound_tx_schedule_interval = 12;
   int64 outbound_tx_schedule_lookahead = 13;
+  string ballot_threshold = 14 [
```

Comment:
I think ballot threshold , can be moved out this struct as well. Considering its not related to chains at all .
We can consider a mapping between tx types , and ballot_threshold in the future , or just assign thresholds while creating ballots which we are doing now

Reply:
That makes sense, I think we can create an issue and group it with the idea of setting a single `min_self_delegation` as well as a general task for observer simplification